Source/ui_qt/PreferenceDefs.h

```diff
@@ -2,4 +2,4 @@
 
 #define PREFERENCE_AUDIO_ENABLEOUTPUT "audio.enableoutput"
 #define PREF_UI_PAUSEWHENFOCUSLOST "ui.pausewhenfocuslost"
-#define PREF_VIDEO_USEVULKAN "video.usevulkan"
+#define PREF_VIDEO_GS "video.gs"
```

Comment:
Please rename this to `PREF_VIDEO_GS_HANDLER` and `video.gshandler`.

Reply:
Also, we should have an `enum` to define the possible values for that preference (currently OpenGL or Vulkan) instead of using 0 and 1 in the code.

Reply:
should the enum go in `CGSHandler`